app/views/design-system/rsd-design-system/design-patterns/trust-details/index.html

```diff
@@ -0,0 +1,149 @@
+                &lt;/li&gt;
+              &lt;/ul&gt;
+            &lt;/div&gt;
+          &lt;/div&gt;
+          &lt;hr class=&quot;govuk-footer__section-break&quot;&gt;
+          &lt;div class=&quot;govuk-footer__meta&quot;&gt;
+            &lt;div class=&quot;govuk-footer__meta-item govuk-footer__meta-item--grow dfe-footer__meta-item--grow&quot;&gt;
+              &lt;h2 class=&quot;govuk-visually-hidden&quot;&gt;Support links&lt;/h2&gt;
+              &lt;ul class=&quot;govuk-footer__inline-list&quot;&gt;
+                &lt;li class=&quot;govuk-footer__inline-list-item&quot;&gt;
+                  &lt;a class=&quot;govuk-footer__link&quot; href=&quot;#&quot;&gt;
+                    Accessibility statement
+                  &lt;/a&gt;
+                &lt;/li&gt;
+                &lt;li class=&quot;govuk-footer__inline-list-item&quot;&gt;
+                  &lt;a class=&quot;govuk-footer__link&quot; href=&quot;#&quot;&gt;
+                    Cookies
+                  &lt;/a&gt;
+                &lt;/li&gt;
+              &lt;/ul&gt;
+            &lt;/div&gt;
+            &lt;div class=&quot;govuk-footer__meta-item&quot;&gt;
+              &lt;a class=&quot;govuk-footer__link govuk-footer__copyright-logo&quot; href=&quot;https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/&quot;&gt;© Crown copyright&lt;/a&gt;
+            &lt;/div&gt;
+          &lt;/div&gt;
+        &lt;/div&gt;
+      &lt;/footer&gt;
+</code>' %}
+
+{% block beforeContent %}{% endblock %}
+
+{% block content %}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        {{ sample.code('/design-system/rsd-design-system/design-patterns/contact-details/example.html', 'Contact details', 'contact-details',
+        htmlCode) }}
+        <br>
+
+        <h2>When to use this component</h2>
+        <p>Use this pattern to display read-only trust details. Provide another page to let users edit the details when required.</p>
+
+        <h2>When not to use this component</h2>
+        <p>Do not use this pattern if your design only has one trust details field. For example, a person's name.</p>
+
+        <h2>How to use this component</h2>
+        <p>If displaying trust details on a standalone page, use the <a href="https://design-system.service.gov.uk/components/summary-list/">summary list component</a>.
+
+        <img src="public/assets/images/trust-summary-list.png" alt class="A summary list with details about a trust.">
+      
+        <p>If displaying trust details alongside other summary lists on a page, use the <a href="https://design-system.service.gov.uk/components/summary-list/#summary-cards">summary card</a> design.</p>
+
+        <img src="public/assets/images/trust-summary-cards.png" alt class="2 summary cards. One is called Trust details, and another is called Reference numbers.">
```

Comment:
You've put alt text in the class attribute. Also make sure the alt text is properly describing the image, and consider using the HTML to render it instead of using an image.

Reply:
Thanks for spotting it. This page is not published yet (no links can access it) so we will fix it later.

Reply:
Thanks for flagging it. We have just removed the entire section for now but we will definitely pay attention to that problem in other parts of the website.